app/components/layout/top-nav.tsx

```diff
@@ -103,7 +111,7 @@ export function TopNav() {
   );
 }
 
-function LogoutButton({ className }: { className?: string }) {
+export function LogoutButton({ className }: { className?: string }) {
```

Comment:
Doesn't look like this export is used anywhere else in the codebase. Is the plan to reuse it in the mobile view later?